runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -650,18 +650,14 @@ else if (shouldNotifyTier)
 
 		if (item.getQuantity() > 1)
 		{
-			notificationStringBuilder.append(" x ").append(item.getQuantity());
-
 
 			if (item.getQuantity() > (int) Character.MAX_VALUE)
 			{
-				notificationStringBuilder.append(" (Lots!)");
```

Comment:
Imo the () format is better than x format as it follow how items are displayed on ground and in menu entries

Reply:
> Imo the () format is better than x format as it follow how items are displayed on ground and in menu entries

True for ground items but menu entries it is x not ()? Unless this is something I have configured.

As said in the other PR for items that already use brackets in their name results in things like `potion (4)(6) ` which looks strange. Maybe ground items should be changed.

Reply:
That was already done on #10526 and it wasn't merged because people didn't like the format

Reply:
I guess there are two pull requests fixing the issue one for each display format so I guess whoever does the merging can decide which PR to merge and which to close.

Reply:
I think it was not merged because noone looked at it except 2 people that commented. I can't recall if I saw it for example.

Reply:
Well it is still there and can be merged.